### PR TITLE
fix: Plugin's getting-started.mdx modified and test added

### LINKED ISSUE #186: `plugin/ecmascript/getting-started`'s code is not working
https://swc.rs/docs/plugin/ecmascript/getting-started
I tried to follow this document but the contents seem to be out of date.

e.g. 
```diff
- e.visit_mut_children_with(self);
+ e.span.visit_mut_children_with(self);

- e.left = Ident::new("kdy1".into(), e.left.span()).into();
+ e.left = Box::new(Ident::new("akfm".into(), e.left.span()).into());
```

### PR DESCRIPTION
**Pull request by AkifumiSato**
- resolve #186 
- Adding a simple example of test

### CODE REVIEW

File: pages/docs/plugin/ecmascript/getting-started.mdx [hunk 1]
```diff
@@ -133,12 +133,12 @@ You can do it like
...
 use swc_core::{
     ast::*,
-    visit::VisitMut,
+    visit::{VisitMut, VisitMutWith},
 };
 
 impl VisitMut for TransformVisitor {
     fn visit_mut_bin_expr(&mut self, e: &mut BinExpr) {
-        e.visit_mut_children_with(self);
+        e.span.visit_mut_children_with(self);
```
Comment (kdy1) [inline pages/docs/plugin/ecmascript/getting-started.mdx:141]: This is wrong

File: pages/docs/plugin/ecmascript/getting-started.mdx [hunk 2]
```diff
@@ -149,15 +149,18 @@ e.g. `visit_mut_ident` for `foo` and `bar` will be called by `e.visit_mut_childr
...
+use swc_core::{
+    ast::*,
+    visit::{VisitMut, VisitMutWith},
+    common::Spanned,
+};
 
 impl VisitMut for TransformVisitor {
     fn visit_mut_bin_expr(&mut self, e: &mut BinExpr) {
-        e.visit_mut_children_with(self);
+        e.span.visit_mut_children_with(self);
```
Comment (kdy1) [inline pages/docs/plugin/ecmascript/getting-started.mdx:160]: Same here

File: pages/docs/plugin/ecmascript/getting-started.mdx (hunk 1 above)
Comment (AkifumiSato) [reply pages/docs/plugin/ecmascript/getting-started.mdx]: fixed.

File: pages/docs/plugin/ecmascript/getting-started.mdx (hunk 2 above)
Comment (AkifumiSato) [reply pages/docs/plugin/ecmascript/getting-started.mdx]: fixed.

**Review by kdy1 — APPROVED** [review]
Thank you!

Can you sign the CLA?

**Comment by AkifumiSato** [comment]
@kdy1 Thanks for the review!
I signed it.

**Pull request merged**